### DISCUSSION TITLE
[Feature] Impersonalization

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [immich-app/immich#13101] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to be able to impersonalize another user as admin. The example use case would be as a parental control, where you could access your children account to check whats happening, why it's taking hundreds of GBs for some reason, without the need of remembering or resetting the password.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is not in scope for Immich, if you need advanced auth features like this you should use oauth.
